docs/RUN.md

```diff
@@ -19,6 +19,47 @@ Now that gdbgui is open, you can interactively run a program with it.
 
 For a list of gdbgui arguments, see the Arguments section below or type gdbgui --help.
 
+### Debugging Rust Programs
+
+`gdbgui` can be used to debug programs written in Rust. Assuming you use [Cargo](https://doc.rust-lang.org/stable/cargo/) to create a new program
+and build it in Debug mode in the standard way:
+
+```
+cargo new myprog
+cd myprog
+cargo build
+```
+
+You can start debugging with a simple
+
+`gdbgui target/debug/myprog`
+
+There are a couple of small difficulties.
+
+Instead of showing your `main` function the initial screen will be blank and `gdbgui` will print `File not found: main`.
```

Comment:
I tested this myself and found a couple things.

1.) this `file not found` error is sort of expected, and can be worked around in a couple ways. See https://stackoverflow.com/q/24664994/2893090.

2.) when I build this on my mac, I get a bunch of warnings like this

> warning /Users/chad/examples/rust/target/debug/deps/hello-486956f9dde465e5.9elsx31vb4it187.rcgu.o': can't open to read symbols: No such file or directory.

It's true, those files don't exist. But `rust/target/debug/deps/hello-486956f9dde465e5` does. So if I run `symbol-file /Users/cssmith/git/gdbgui/examples/rust/target/debug/deps/hello-486956f9dde465e5`, gdb successfully loads the symbol table and `hello::main` successfully inserts a breakpoint and opens the src/main.rs file.